testsuite/MDAnalysisTests/analysis/test_rdf.py

```diff
@@ -93,6 +93,12 @@ def test_exclusion(sels):
     rdf = InterRDF(s1, s2, exclusion_block=(1, 2)).run()
     assert rdf.results.count.sum() == 4
 
+def test_ignore_same_residues(sels):
```

Comment:
Sorry to be a pain but we should test all covered attributes?  ie resid, ....

Reply:
And maybe an unsupported argument as well, to solve the codecov warning above?

Reply:
Could you point me to some appropriate test data for segindices and chainids? In the example I am using the segindices is identical for all residues and chainids aren't present.